### DISCUSSION TITLE
Multi iteration PSF fitting

### DISCUSSION THREAD

**Discussion by EranOfek** [EranOfek/AstroPack#61] [Ideas]
after PSF fitting we need functions for:
1. create an image with PSFs at at list of [X,Y,Flux]
2. Subtract this image from the original image
3. use this for a mutli-iteration PSF fitting
(Noam Segev is writing the functions)

**Comment by agioffe** [comment]
A first approach: several tools in **imUtil.art.** (**imUtil.art.createSourceCube**, **imUtil.art.addSources** and others, most of the functionality tested in **imUtil.art.unitTest**) and a higher-level function **imProc.art.injectSources**
